packages/preview/muw-touying/0.1.0/README.md

```diff
@@ -0,0 +1,61 @@
+# Typst/Touying Template for MedUni Wien
+
+This is a template built on [`Touying`](https://github.com/touying-typ/touying) to create presentation slides that capture the design of the official PowerPoint templates following Medizinischen Universität Wien [Corporate Design](https://www.meduniwien.ac.at/web/en/studierende/service-center/meduni-wien-vorlagen/).
+
+## Disclaimer
+
+*This theme is __unofficial__. It is __NOT__ affiliated with Medizinischen Universität Wien (Medical University of Vienna). The logo is the intellectual property of the Medical University of Vienna and is subject to copyright. Users are advised to check the official Styleguide before using this template.*
+
+## Example
+
+See [examples](examples) for more details.
+
+## Usage
+
+Use the following code as a starting point to create your own slides.
+
+```typst
+#import "@preview/touying:0.6.1": *
+#import "lib.typ": *
```

Comment:
This line should show how to import the package from Typst Universe, with a `@preview/...` import.

Reply:
Thanks! I’ve addressed this in the latest commit.